README.md

```diff
@@ -37,7 +37,6 @@ Tekton defines a number of [Kubernetes custom resources](https://kubernetes.io/d
 The custom resources needed to define a pipeline are listed below:
 * `Task`: a reusable, loosely coupled number of steps that perform a specific task (e.g. building a container image)
 * `Pipeline`: the definition of the pipeline and the `Tasks` that it should perform
-* `PipelineResource`: inputs (e.g. git repository) and outputs (e.g. image registry) to and out of a pipeline or task
```

Comment:
As we removed `PipelineResource` and used `Workspace` i feel its good to provide the info about workspace bit.

Reply:
I specified that down below. Here we are only mentioning CRs.